### ISSUE TITLE
The complete content of a list item which contains nested HTML doesn't get announced as one complete chunk of content

### ISSUE THREAD

**Issue by fstorr** [FreedomScientific/VFO-standards-support#5]
### Summary

If a list item contains nested HTML, JAWS 18 won't announce it as one "chunk" of content. Instead, it only announces the content of the first nested element, resulting in the arrow keys being needed to discover the other nested items.
 
### Example:
1. Go to: https://codepen.io/fstorr/full/jGZGBa/
2. Use the <kbd>i</kbd> key to find the next list item.
3. JAWS 18 will announce the content of the first nested element, for example the visually-hidden "completed step:" `<span>` element, and then stop. 

### Expected result

I would expect JAWS to announce the full content of the list item as NVDA 2017.3 does.

### Actual result

JAWS 18 only announces the content of the first nested HTML element.

### Example

Test case on CodePen: https://codepen.io/fstorr/full/jGZGBa/

#### JAWS version and build number

JAWS 18.0.4321
 
#### Operating System and version

Windows 10 Enterprise 64-bit

#### Browser and version:

Chrome 61, IE11, Firefox 56.0

**Comment by LJWatson** [comment]
Confirmed in Firefox 56, Chrome 61, and IE11 with the Jaws 2018 public beta. Jaws does not have the same issue in Edge though.

**Comment by fstorr** [comment]
I tried Edge, even with Edge Support enabled in JAWS, and couldn't get any of the quick navigation keys working. Maybe it's my version of Edge, which is: EdgeHTML 14.14393.

**Comment by stevefaulkner** [comment]
Filed internally as 
Bug #97003 - The complete content of a list item which contains nested HTML doesn't get announced as one complete chunk of content

**Comment by frex65** [comment]
This bug is quite nuanced. If the HTML within the list item is broken up by JAWS into more than one line (so if <div> or <a href="...."> elements are used), only the first element is spoken. But if, for example, the list item only contains <span> elements, JAWS speaks all of the text.

**Comment by TPGJRogers** [comment]
While the amount of information read did and does vary based on the HTML within that list item, as described in the issue; documentation for the I command indicates it will move to, not read, the next list item. Closing the issue.
